src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -107,9 +108,12 @@ public string ItemType
             {
                 ItemTypeVisibility = true;
                 SetProperty(ref itemType, value);
+                OnPropertyChanged(nameof(IsFileFolder));
             }
         }
 
+        public bool IsFileFolder => ItemType != "File folder";
```

Comment:
Note that file type string is localized so this check only works is system language is English. Pushed a commit to change this.